src/wp-admin/load-scripts.php

```diff
@@ -58,16 +56,5 @@
 header('Expires: ' . gmdate( "D, d M Y H:i:s", time() + $expires_offset ) . ' GMT');
 header("Cache-Control: public, max-age=$expires_offset");
 
-if ( $compress && ! ini_get('zlib.output_compression') && 'ob_gzhandler' != ini_get('output_handler') && isset($_SERVER['HTTP_ACCEPT_ENCODING']) ) {
```

Comment:
this is part of the improvement of 5.0 where they removed this code that improve the performance because the compression of pages is made by web server and not php.

Reply:
better add something for that in release notes